### PR TITLE
Refs #12723 - Fix content count factory for debian content

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

**Comment by jlsherrill** [comment]
looks like there is a small test breakage (ignore the react test failure)

**Comment by mdellweg** [comment]
Oh yeah, that was a trivial oversight.

File: app/views/foreman/smart_proxies/_content_tab.html.erb [hunk 1]
```diff
@@ -8,9 +8,11 @@
       <th translate>Last Published</th>
       <th translate>Hosts</th>
       <th translate>Products</th>
+      <th translate>Apt repos</th>
       <th translate>Yum repos</th>
       <th translate>Container Image repos</th>
-      <th translate>Packages</th>
+      <th translate>Debian Packages</th>
+      <th translate>Redhat Packages</th>
```
Comment (jturel) [inline app/views/foreman/smart_proxies/_content_tab.html.erb]: 'Red Hat Packages' :)

Comment (jlsherrill) [reply app/views/foreman/smart_proxies/_content_tab.html.erb]: I'd actually prefer 'RPM Packages'   but good job @jturel for keeping brand in mind ;)

Comment (jturel) [reply app/views/foreman/smart_proxies/_content_tab.html.erb]: yeah good call on 'RPM Packages'!

Comment (mdellweg) [reply app/views/foreman/smart_proxies/_content_tab.html.erb]: RPM Packages it is... ;)
Also changed Debian (the distribution) to Deb (the file format).

**Pull request merged**